crates/nu-path/src/pwd_per_drive.rs

```diff
@@ -0,0 +1,366 @@
+use std::path::{Path, PathBuf};
+
+cfg_if::cfg_if! { if #[cfg(windows)] {
+
+
+struct Drive2PWDmap {
```

Comment:
As far as naming goes, this is very unusual for Rust. I would go with `DriveToPwdMap` or `PerDriveCurrentDirectoryMap`

Reply:
rename to 'DriveToPwdMap'